SimplenoteIntents/IntentHandler.swift

```diff
@@ -8,15 +9,28 @@ class IntentHandler: INExtension {
 
 extension IntentHandler: NoteWidgetIntentHandling {
     func provideNoteOptionsCollection(for intent: NoteWidgetIntent, with completion: @escaping (INObjectCollection<WidgetNote>?, Error?) -> Void) {
-        // This is placeholder code to confirm the dynamic intent selection is working
-        // TODO: add logic to fetch the available notes for account
+        // NOTE: This core data implementation is just to prove that the shared database is accessible and is not intended to be used in production.
```

Comment:
(As discussed on Slack!) let's drop these changes from this PR, since it's PoC code.

Thank you sir!